diff --git a/node_modules/@chakra-ui/theme/dist/types/index.d.ts b/node_modules/@chakra-ui/theme/dist/types/index.d.ts
index c4e8c3a..056bd0d 100644
--- a/node_modules/@chakra-ui/theme/dist/types/index.d.ts
+++ b/node_modules/@chakra-ui/theme/dist/types/index.d.ts
@@ -2998,6 +2998,23 @@ export declare const theme: {
         full: string;
     };
     colors: {
+        primary: {
+            regular: string;
+            light: string;
+        };
+        secondary: {
+            regular: string;
+            light: string;
+        };
+        text: {
+            dark: string;
+            light: string;
+        };
+        bg: {
+            dark: string;
+            light: string;
+        };
+        warning: string;
         transparent: string;
         current: string;
         black: string;
