diff --git a/node_modules/@chakra-ui/theme/dist/types/foundations/index.d.ts b/node_modules/@chakra-ui/theme/dist/types/foundations/index.d.ts
index 05c10f1..32d8541 100644
--- a/node_modules/@chakra-ui/theme/dist/types/foundations/index.d.ts
+++ b/node_modules/@chakra-ui/theme/dist/types/foundations/index.d.ts
@@ -215,6 +215,19 @@ declare const foundations: {
         full: string;
     };
     colors: {
+        primary: {
+            regular: string;
+            light: string;
+        };
+        secondary: {
+            regular: string;
+            light: string;
+        };
+        text: {
+            primary: string;
+            light: string;
+        };
+        warning: string;
         transparent: string;
         current: string;
         black: string;
