diff --git a/node_modules/@chakra-ui/theme/dist/types/index.d.ts b/node_modules/@chakra-ui/theme/dist/types/index.d.ts
index c4e8c3a..a3a5d47 100644
--- a/node_modules/@chakra-ui/theme/dist/types/index.d.ts
+++ b/node_modules/@chakra-ui/theme/dist/types/index.d.ts
@@ -2998,6 +2998,31 @@ export declare const theme: {
         full: string;
     };
     colors: {
+        primary: {
+            50: string;
+            100: string;
+            200: string;
+            300: string;
+            400: string;
+            500: string;
+            600: string;
+            700: string;
+            800: string;
+            900: string;
+          },
+        secondary: {
+            200: string;
+            300: string;
+        },
+        text: {
+            dark: string;
+            light: string;
+        },
+        bg: {
+            dark: string;
+            light: string;
+        },
+        warning: string;
         transparent: string;
         current: string;
         black: string;
