diff --git a/node_modules/@chakra-ui/theme/dist/types/index.d.ts b/node_modules/@chakra-ui/theme/dist/types/index.d.ts
index 6813f5c..0f0e06c 100644
--- a/node_modules/@chakra-ui/theme/dist/types/index.d.ts
+++ b/node_modules/@chakra-ui/theme/dist/types/index.d.ts
@@ -2835,6 +2835,7 @@ export declare const theme: {
         inner: string;
         none: string;
         "dark-lg": string;
+        dark: string;
     };
     space: {
         px: string;
@@ -2991,6 +2992,31 @@ export declare const theme: {
         full: string;
     };
     colors: {
+        primary: {
+            50: string;
+            100: string;
+            200: string;
+            300: string;
+            400: string;
+            500: string;
+            600: string;
+            700: string;
+            800: string;
+            900: string;
+          },
+        secondary: {
+            200: string;
+            300: string;
+        },
+        text: {
+            dark: string;
+            light: string;
+        },
+        bg: {
+            dark: string;
+            light: string;
+        },
+        warning: string;
         transparent: string;
         current: string;
         black: string;
